src/App.js

```diff
@@ -1,25 +1,80 @@
-import logo from './logo.svg';
 import './App.css';
```

Comment:
I really like how your code is set up, It is really easy to read and understand. I would recommend adding your deploy URL to your page (like we did when we used GitHub pages) so other people can see what your page looks like.